datastore/google/cloud/datastore/key.py

```diff
@@ -324,7 +325,8 @@ def from_legacy_urlsafe(cls, urlsafe):
         This is intended to work with the "legacy" representation of a
         datastore "Key" used within Google App Engine (a so-called
         "Reference"). This assumes that ``urlsafe`` was created within an App
-        Engine app via something like ``ndb.Key(...).urlsafe()``.
+        Engine app via something like ``ndb.Key(...).urlsafe()``. The base64
+        encoded values will have padding removed.
```

Comment:
Here the comment is inaccurate, and probably not needed? The base64 encode values are the **input** rather than the output. And this method will work with **both** padded and unpadded values (and even incorrectly padded values, so a certain point).

I think you can probably just remove this comment?